### DISCUSSION TITLE
How to allow boss overhealing?

### DISCUSSION THREAD

**Discussion by naydef** [Batfoxkid/Freak-Fortress-2-Rewrite#106] [Q&A] [answered]
Basically what happens when you give overheal to a boss is that the overheal decays extremely fast.

So I wonder whether FF2R somehow purposely makes overheal decay very fast or it's something game inherent? I tried to influence this behavior with attributes (with indexes 12, 13 and 14), but there was no noticeable change of behavior.

**Comment by Batfoxkid** [comment, marked as answer]
This is a effect caused by how FF2R handles max health versus other FF2 versions. Instead of using `SDKHook_GetMaxHealth` to set the max  health, FF2R instead sets it via an attribute which makes overheal correctly (or in this case not wanted) decay at rate compared to max health.
